src/commands/utility/vote-points.js

```diff
@@ -17,11 +17,13 @@ module.exports = {
     .addStringOption((option) =>
       option
         .setName('point-type')
-        .setDescription('Select the type of points to award')
+        .setDescription(translateLanguage('Select the type of points to award'))
```

Comment:
oh this translation will not work ;(

You should add the translation on en.json and es.json files, then use here the property  that you created on the json, same that if you are using i18n

Here is the translation folder where the jsons are https://github.com/TeamNovaSoft/discord-bot/tree/main/src/languages

Reply:
@heliomar-pena check